malasakit-django/pcari/apps.py

```diff
@@ -1,27 +0,0 @@
-"""
```

Comment:
Put this file back. It enables signals, which is critical for deletion to work correctly.